docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Logging_Formatters.adoc

```diff
@@ -0,0 +1,322 @@
+= Logging Formatters
+:author:            James R. Perkins
+:email:             jperkins@redhat.com
+:idprefix:
+:idseparator:       -
+:oracle-javadoc:     https://docs.oracle.com/javase/8/docs/api/java
+
+Formatters are used to format a log message. A formatter can be assigned to a logging handler.
+
+The logging subsystem includes 4 types of handlers:
+
+* <<json-formatter>>
+* <<pattern-formatter>>
+* <<xml-formatter>>
+* <<custom-formatter>>
+
+== JSON Formatter
+
+A formatter used to format log messages in JSON.
+
+=== Examples
+
+.Simple JSON Formatter
+----
+/subsystem=logging/json-formatter=json:add(pretty-print=true, exception-output-type=formatted)
+----
+
+.Logstash Formatter
+----
+/subsystem=logging/json-formatter=logstash:add(exception-output-type=formatted, key-overrides=[timestamp="@timesamp"],
```

Comment:
@jamezp  `@timesamp` is probably a typo?

Reply:
Yup, all fixed now.